app/assets/javascripts/editor.js

```diff
@@ -174,10 +174,13 @@ class Editor {
     const previewBtn = $("#toggle-preview-button-" + this.commentFormID);
     const formIdPrefix = this.isSingleFormPage ? "#form-body-" : "#comment-form-body-";
     const commentFormBody = $(formIdPrefix + this.commentFormID);
+    // const toolbar = $('.btn-toolbar')[0];
+    const toolbar = commentFormBody.parentElement.children[3]
```

Comment:
Hi @KarishmaVanwari, I don’t think you’re actually grabbing the `toolbar`. The `toolbar` isn’t a part of the `commentFormBody`.

Reply:
You should use the logic used in lines 175 and 176 to grab the `toolbar` but remove the `body` attached to it so that it applies to the entire form and not just the `body`.

Reply:
Okay @TildaDares , I'll try it.
I am unable to post a comment on the local environment, because of which I am unable to verify if my changes do the work required. Could you please let me know why am I unable to post a comment @TildaDares ? It only said 'an error occurred' on posting a comment.

Reply:
Can I see a screenshot of your screen when this error occurs?

Reply:
![image](https://user-images.githubusercontent.com/78212650/144229580-ebf95876-7a3c-4efc-b1a8-3e94a3163786.png)
 here it is @TildaDares

Reply:
Maybe you should try using an admin account instead. Can you also paste the console errors here?

Reply:
@KarishmaVanwari I think the problem you're having is because of the code you added. Try removing the code and see what happens.

Reply:
@TildaDares I tried by removing my piece of code and also by keeping it. It still shows the same error. But when I refreshed the page, I could see the comments but its gives the count as zero. Is this what I am supposed to see?
![image](https://user-images.githubusercontent.com/78212650/144232929-cbbde72a-ecbd-4f16-9b7a-8fe16ddd1aad.png)

Reply:
Unapproved comments aren’t counted.

Reply:
Try using an admin account so you don’t have to go through the whole comment approval process

Reply:
Okay, thank you @TildaDares